drivers/sensor/ina23x/Kconfig

```diff
@@ -26,6 +26,14 @@ config INA237
 	help
 	  Enable driver for INA237.
 
+config INA237_VSHUNT
+	bool "INA237"
+	default y
+	depends on DT_HAS_TI_INA237_ENABLED
+	help
+	  Enable VShunt measurement for INA237.  This requires an extra I2C
+	  read when enabled and
```

Comment:
and what?

Reply:
Ooops, thanks for catching that.  Will get it updated today.

Reply:
Updated the description and pushed the commit with just the change.  Will push a rebase next.